targets/esp8266/source/jerry_extapi.c

```diff
@@ -97,27 +96,27 @@ DELCARE_HANDLER(gpio_dir) {
     return false;
   }
 
-  port = (int)JS_VALUE_TO_NUMBER (&args_p[0]);
-  value = (int)JS_VALUE_TO_NUMBER (&args_p[1]);
+  port = (int)jerry_get_number_value (args_p[0]);
```

Comment:
minor style issue: missing space after type cast
There some of this issue, please fix them all.